src/main/java/duke/Command.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+public class Command {
```

Comment:
The name for this class may not be that clear as you are not really representing a command but rather using the class to parse commands. Is there a better name you can use?